executor/insert.go

```diff
@@ -149,9 +149,9 @@ func prefetchDataCache(ctx context.Context, txn kv.Transaction, rows []toBeCheck
 	return prefetchConflictedOldRows(ctx, txn, rows, values)
 }
 
-// updateDupRowNew updates a duplicate row to a new row.
-func (e *InsertExec) updateDupRowNew(ctx context.Context, txn kv.Transaction, row toBeCheckedRow, handle int64, onDuplicate []*expression.Assignment) error {
-	oldRow, err := e.getOldRowNew(ctx, e.ctx, txn, row.t, handle, e.GenExprs)
```

Comment:
so `batchChecker) getOldRowNew` can be remove now~?

Reply:
Not now, the `replace ...` is still using it.
I'll remove it later...

Reply:
https://github.com/pingcap/tidb/blob/dd11b55699c615377a481667eef76b5fa5ae03a7/executor/batch_checker.go#L297 this method can not find any usage :D